src/stores/wordpress-versions-slice.ts

```diff
@@ -0,0 +1,104 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import * as Sentry from '@sentry/electron/renderer';
+import { z, ZodError } from 'zod';
+
+const MINIMUM_WORDPRESS_VERSION = '5.9.9';
+
+const wordPressOfferSchema = z.object( {
+	version: z.string(),
+	response: z.enum( [ 'autoupdate', 'upgrade' ] ),
+} );
+
+const wordPressApiResponseSchema = z.object( {
+	offers: z.array( wordPressOfferSchema ),
+} );
+
+interface WordPressVersion {
+	version: string;
+	isBeta: boolean;
+	name: string;
+}
+
+interface WordPressVersionsState {
+	versions: WordPressVersion[];
+	status: 'idle' | 'loading' | 'succeeded' | 'failed';
+	error: string | null;
+}
+
+const initialState: WordPressVersionsState = {
+	versions: [],
+	status: 'idle',
+	error: null,
+};
```

Comment:
Optional nit, but I'd move this part closer to the `createSlice` call. I don't think there's a universal standard, but that's what I did for the chat slice, and I think it makes sense.